LICENSE

```diff
@@ -1,6 +1,6 @@
 OnionShare
 
-Copyright © 2016
+Copyright © 2018
```

Comment:
@micahflee Can I recommend we add a single line under the Copyright here saying 'Note: further licenses relating to Onionshare's use of third party software can be found under install/licenses/'

This makes it clear at the top level that there are other licenses deeper in the source code.

Reply:
Good idea!